.github/ISSUE_TEMPLATE/documentation_request.md

```diff
@@ -0,0 +1,40 @@
+---
+name: Documentation request
+about: Missing documentation or suggest an improvement
+
+---
+
+<!--
+Thanks for submitting this documentation request. Please fill the template below,
+otherwise we will not be able to process this request.
+
+Open an issue with this template if you think documentation regarding a certain aspect of the stack is lacking or
```

Comment:
- We never say "the stack", it's always "The Things Stack"
- Respect max line length of ±80